packages/bootstrap/scss/_variables.scss

```diff
@@ -138,8 +138,9 @@ $toolbar-padding-x: $table-cell-padding / 2 !default;
 /// @group toolbar
 $toolbar-padding-y: $table-cell-padding / 2 !default;
 
-$toolbar-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2} + 2px;
-$toolbar-inner-calc-size: #{$button-calc-size} + #{$toolbar-padding-y * 2};
+$toolbar-button-padding-x: $button-padding-x !default;
+$toolbar-button-padding-y: $button-padding-y !default;
```

Comment:
These are the default values in the Default theme. No need to re-state them here.